src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -138,11 +138,11 @@ module Types = struct
   let sync_status : ('context, Sync_status.t option) typ =
     enum "SyncStatus" ~doc:"Sync status of daemon"
       ~values:
-        [ enum_value "BOOTSTRAP" ~value:`Bootstrap
-        ; enum_value "SYNCED" ~value:`Synced
-        ; enum_value "OFFLINE" ~value:`Offline
-        ; enum_value "CONNECTING" ~value:`Connecting
-        ; enum_value "LISTENING" ~value:`Listening ]
+        (List.init (Sync_status.max + 1) ~f:(fun i ->
+             let status = Option.value_exn (Sync_status.of_enum i) in
+             enum_value
+               (String.map ~f:Char.uppercase @@ Sync_status.to_string status)
```

Comment:
String.uppercase?

Reply:
`String.uppercase` just uppercase the first character of a string. We would like to have all.